hello-apl/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, gnuapl }:
+
+stdenv.mkDerivation {
+  pname = "hello-apl";
+  version = "0.1.0";
+
+  nativeBuildInputs = [ gnuapl ];
```

Comment:
I'm not entirely sure, but I _think_ this should be a `buildInputs` since you need it to run the script.

`nativeBuildInputs` are for things that you _only_ need during the build.